docker/publish-public-images.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+set -ex
+
+project=$1
+
+make -C ../hail python/hail/hail_pip_version
+cp ../hail/python/hail/hail_pip_version hail/hail_pip_version
+hail_pip_version=$(cat hail/hail_pip_version)
+
+build_and_push() {
+    name=$1
+    base=$2
+
+    docker=hailgenetics/$name:$hail_pip_version
+    google=gcr.io/$project/$name:$hail_pip_version
+    latest=gcr.io/$project/$name:latest
+
+    docker pull $latest || true
+    docker build \
+           $name/ \
+           -f $name/Dockerfile.out \
+           -t $docker \
+           -t $google \
+           -t $latest \
+           --cache-from $latest,$base
+    docker push $docker
+    docker push $google
+    docker push $latest
+}
+
+python3 ../ci/jinja2_render.py '{"hail_ubuntu_image":{"image":"hail-ubuntu"}}' hail/Dockerfile.base hail/Dockerfile.base.out
```

Comment:
Maybe this was a different PR, but I think the Dockerfile name is not correct here.